_posts/2019-09-30-whats-new-in-strimzi-0.14.0.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: post
+title:  "What's new in Strimzi 0.14.0"
+date: 2019-09-30
+author: jakub_scholz
+---
+
+We are delighted to announce the availability of Strimzi 0.14.0 release.
+It is our first release as a CNCF Sandbox project and it is a big release with lot of new features.
+This blog post will give you an overview of the new features and will be followed in next days with more detailed blog posts dedicated to each of them.
+
+<!--more-->
+
+## OAuth Authentication
+
+Strimzi already supported authentication and authorization before the 0.14.0 release.
+We even have a dedicated operator for managing users - which we conveniently call _User Operator_.
+It supports authentication using TLS client certificates and usernames and passwords.
+It also supports authorization using the SimpleACLAuthorizer which is part of Apache Kafka.
+All of this is managed using the `KafkaUser` custom resources directly inside your Kubernetes cluster.
+The User Operator works very well in many situations and there are many users who like like it.
+But there are many situations when you do not want to manage your Kafka users separately from all other users.
+
+One of the reasons might be that you want to have all users managed inside a single system.
+That gives you much better overview of which users exist and what can they do. 
+You can see it all in one place.
+Most systems are also not based only around Apache Kafka.
+Often single applications needs to use Kafka as well as for example some HTTP or other APIs.
+And ideally it should be able to use a single identity for everything.
+
+Kafka already supports authentication using SASL OAUTHBEARER mechanism. 
+This mechanism allows clients to OAuth tokens obtained from and OAuth 2.0 authorization server and use them to authenticate against the Kafka broker.
+And in the 0.14.0 release we are adding support for it also in Strimzi.
+To enable the support, we developed our own [callback library](https://github.com/strimzi/strimzi-kafka-oauth) responsible for obtaining and validating the OAuth tokens.
+This library is not used only by the Kafka brokers.
+It is also used in all the other components which Strimzi supports and which connect to the Kafka brokers as clients such as Connect, Mirror Maker or our HTTP Bridge.
+This library is also available in [Maven repositories](https://search.maven.org/artifact/io.strimzi/oauth/0.1.0/pom), so that you can use it in your own clients using the Kafka Producer, Consumer and Streams APIs.
+
+In 0.14.0 we have just started with OAuth support.
+We did extensive testing with selected OAuth 2.0 authorization servers such as [Keycloak](https://www.keycloak.org/).
```

Comment:
We should mention hydra too.

Reply:
I do not think we really tested it as working. Unless I missed something on Friday.